### DISCUSSION TITLE
Flows naming conventions

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#15657] [Feature Requests]
@directus/team 

Currently, there seem to be multiple naming conventions within Flows:

- In event hooks, we use the terms `Filter (Blocking)` and `Action (Non-Blocking)`. (Which feels like two sets of terms)
- In webhooks and manual, we have `Asynchronous` with a toggle switch to enable/disable.

Are these essentially the same? If not what's the difference?
If they are pretty much the same, is it possible to converge on one set of terms?

**Comment by br41nslug** [comment]
The terms `Blocking`/`non-blocking` are generally the same as `synchronous`/`asynchronous`. Having said that im not sure why but renaming the event hooks to `Filter (Synchronous)` and `Action (Asynchronous)` doesnt feel right for some reason 🤔

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
